docs/conf.py

```diff
@@ -143,7 +143,7 @@ def isHTTPLink(uri):
     return uri.startswith('http://') or uri.startswith('https://')
 
 def isMDFileLink(uri):
-    return uri.endswith('.md')
+    return uri.endswith('.md') or uri.find('.md#') != -1
```

Comment:
However, just a bit more pythonic way: `if '.md#' in uri`

Reply:
Tnaks, I'll update.